pkg/cmd/experimental/syncgroups/schema2/config.go

```diff
@@ -0,0 +1,16 @@
+package schema2
```

Comment:
Move this entire package out of this pull.  I'm not going to review it until after schema1 is done.